spot-setup/README.md

```diff
@@ -28,6 +32,11 @@ This file also contains sources desired to be installed as part of Apache Spot,
 
 To read more about these variables, please review the [wiki] (https://github.com/Open-Network-Insight/open-network-insight/wiki/Edit%20Solution%20Configuration).
 
+## Install
+
+1. run `sudo ./install.sh`
+2. run `hdfs_setup.sh`
```

Comment:
I agree, spot.conf could be added very easily.
However i was thinking i might do move that to /opt/spot/spot.conf instead. this requires changing each components file path which should be fairly simple.

In regards to the discussion,
It would be simple to have a `install-all.sh` script that ran everything,

#!/bin/bash

source ./install.sh
source ../spot-ingest/install.sh
source ../spot-ml/install.sh
source ../spot-oa/install.sh
and then a user can make the choice if they are landing individual components elsewhere.